### PR TITLE
Add Circuit Operations page

### PR DESCRIPTION
**Pull request by oliverfrye**
I noticed a lot of tower controllers (not the least myself!) struggling with understanding circuit operations and especially what they can and can't do to separate / segregate traffic (especially in the absence of an overlying TMA controller). My hope with this page is to help S2 controllers such as myself to deal with VFR and circuit operations in a realistic, yet practical way.

I tried to summarise relevant techniques and phraseology from the existing VATPAC SOPS, the VATPAC Academy, MATS, and ERSA.

I'd appreciate any and all feedback 🙂

### CODE REVIEW

**Review by alexd-332 — APPROVED** [review]

**Review by mattkelly4 — COMMENTED** [review]
This looks great Oliver, thanks for putting all this together.  Just a few suggestions from me, also just wondering whether you've added breaklines to condense long lines into several shorter lines?  It'll probably be more responsive on a variety of screen sizes if everything just runs the full width and the browser wraps text as required.

My only other suggestion would be potentially merging the **Approach Instructions** and **Delaying Aircraft** headings together, since they both cover similar content.  Maybe they could both fall under **Sequencing**?

File: docs/controller-skills/CircuitOperations.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+at any aerodrome, including major Class C aerodromes.
+
+## Clearance
+
+At a Class C aerodrome, provide an explicit clearance to operate in the circuit
+area, alongside an altitude and a discrete squawk code.
+
+!!! example
+    *ABC is a VFR aircraft on the ground at Canberra that intends to conduct circuit training.*  
+    **ABC:** "Canberra Ground, ABC, request clearance for circuits, with information L"
```
[the remark this answers is not in the document]
Comment (oliverfrye) [reply docs/controller-skills/CircuitOperations.md:18]: Not required for VFR at all, or not required in this particular call?

[the remark this answers is not in the document]
Comment (mattkelly4) [reply docs/controller-skills/CircuitOperations.md:18]: > Not required for VFR at all, or not required in this particular call?

ATIS info isn't required when requesting airways clearance for VFR or IFR, generally just when requesting taxi.  If you're requesting taxi and airways clearance together, then you'd include it.

[the remark this answers is not in the document]
Comment (oliverfrye) [reply docs/controller-skills/CircuitOperations.md:18]: Ah, I get it now! Thanks 🙂

File: docs/controller-skills/CircuitOperations.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    **DEF:** "Cleared to land, DEF"
+
+### Change of Runway
+
+An **ADC** may change the runway for an aircraft in the circuit, so as to
+separate circuit traffic from non-circuit traffic, or to segregate aircraft with
+different performances for better traffic flow.
+
+!!! note
+    Do not offer a change of runway to an aircraft below `A005` on final.
```
[the remark this answers is not in the document]
Comment (oliverfrye) [reply docs/controller-skills/CircuitOperations.md]: Good catch, thank you!

File: docs/controller-skills/CircuitOperations.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+!!! example
+    *DEF, a Cessna 172, is on a left downwind on runway 30 at Canberra, while UEM, a Baron, is joining left base.*
+    **DEF:** "DEF, downwind, full-stop"  
+    **CB ADC:** "DEF, traffic is a Baron joining base to land, orbit right, report orbit complete"  
+    **DEF:** "Orbit right, wilco, DEF"  
+    **CB ADC:** "UEM, traffic is a Cessna 172 orbiting downwind to give way to you, A030, cleared to land"  
+    **UEM:** "Traffic in sight, cleared to land, UEM"  
+    **DEF:** "DEF, orbit complete"  
+    **CB ADC:** "DEF"
```
[the remark this answers is not in the document]
Comment (oliverfrye) [reply docs/controller-skills/CircuitOperations.md]: Agreed - I have used your example mostly, just added comms to ensure that the traffic is sighted for consistency with the earlier example.

File: docs/controller-skills/CircuitOperations.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+    **ABC:** "Extend downwind, traffic in sight, ABC"  
+    **SY ADC:** "QFA1254, traffic is a Citation on a left downwind, A015"  
+    **QFA1254:** "QFA1254, looking"  
+    *Some time later, after QFA1254 is no longer a factor and wake turbulence separation has been assured.*  
+    **SY ADC:** "ABC, continue approach, report final"  
+    **ABC:** "Wilco, ABC"  
+    *At 4nm from touchdown.*  
+    **ABC:** "ABC, final"  
+    **SY ADC:** "ABC, cleared touch and go"  
+    **ABC:** "Cleared touch and go, ABC"
```
[the remark this answers is not in the document]
Comment (oliverfrye) [reply docs/controller-skills/CircuitOperations.md]: This makes good sense to me - I've added a little bit of text above this to explain as well.

File: docs/controller-skills/CircuitOperations.md [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+!!! example
+    **ABC:** "ABC, downwind, request the option"  
+    **AF ADC:** "ABC, make full stop due to traffic, cleared to land"  
+    **ABC:** "Cleared to land, ABC"
+
+## Sequencing
+
+In Class C airspace, separation is required between IFR and VFR aircraft. In
+Class D airspace, separation is only required between IFR aircraft.
+Nevertheless, controllers should still make all efforts to ensure an expiditious
```
[the remark this answers is not in the document]
Comment (oliverfrye) [reply docs/controller-skills/CircuitOperations.md]: Haha I just noticed this a moment ago too - fixed!

File: docs/controller-skills/CircuitOperations.md [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+
+An **ADC** may issue instructions for a pilot to make a longer or shorter
+approach, or to explicitly extend their downwind leg, so as to either pass in
+front of or behind another aircraft.
+
+!!! example
+    *ABC is downwind on runway 16L at Sydney while QFA1254 is established on a 3 mile final for the same runway.*
+    **ABC:** "ABC, downwind, request touch and go"  
+    **SY ADC:** "ABC, traffic is a Boeing 737 on a 3 mile final, extend downwind"  
+    **ABC:** "Extend downwind, traffic in sight, ABC"
```
[the remark this answers is not in the document]
Comment (oliverfrye) [reply docs/controller-skills/CircuitOperations.md]: I've removed the use of the phrase "extend downwind" in favour of "follow" in accordance with Matt K's earlier comment above - I think that covers your concern as well.

File: docs/controller-skills/CircuitOperations.md [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+front of or behind another aircraft.
+
+!!! example
+    *ABC is downwind on runway 16L at Sydney while QFA1254 is established on a 3 mile final for the same runway.*
+    **ABC:** "ABC, downwind, request touch and go"  
+    **SY ADC:** "ABC, traffic is a Boeing 737 on a 3 mile final, extend downwind"  
+    **ABC:** "Extend downwind, traffic in sight, ABC"  
+    **SY ADC:** "QFA1254, traffic is a Citation on a left downwind, A015"  
+    **QFA1254:** "QFA1254, looking"  
+    *Some time later, after QFA1254 is no longer a factor and wake turbulence separation has been assured.*
```
[the remark this answers is not in the document]
Comment (oliverfrye) [reply docs/controller-skills/CircuitOperations.md]: Added a bit regarding wake turbulence standard 👍

**Comment by oliverfrye** [comment]
Thanks @mattkelly4 for the in-depth review! 🙂 

> []snip] just wondering whether you've added breaklines to condense long lines into several shorter lines? It'll probably be more responsive on a variety of screen sizes if everything just runs the full width and the browser wraps text as required.

The breaklines don't show up as new lines on any screen size. In Markdown, newlines are only added explicitly if two spaces are left at the end of a line (as can be seen in the example blocks of this page), or if an empty line is placed between two paragraphs (which creates the desired paragraph space). I have wrapped to 80 characters for my editing comfort (it makes it much easier for me personally), but of course I'm happy to change or remove the wrapping based on any existing conventions in the project .

> My only other suggestion would be potentially merging the **Approach Instructions** and **Delaying Aircraft** headings together, since they both cover similar content. Maybe they could both fall under **Sequencing**?

I was intending to add further information (perhaps with diagrams) in a future PR to each of these sections, so I think they're warranted (and help to break up the page), but happy to remove if you still think they're unnecessary.

**Comment by oliverfrye** [comment]
Thanks for the feedback @LeGiraffe! 🙂

**Comment by mattkelly4** [comment]
> Thanks @mattkelly4 for the in-depth review! 🙂
> 
> > []snip] just wondering whether you've added breaklines to condense long lines into several shorter lines? It'll probably be more responsive on a variety of screen sizes if everything just runs the full width and the browser wraps text as required.
> 
> The breaklines don't show up as new lines on any screen size. In Markdown, newlines are only added explicitly if two spaces are left at the end of a line (as can be seen in the example blocks of this page), or if an empty line is placed between two paragraphs (which creates the desired paragraph space). I have wrapped to 80 characters for my editing comfort (it makes it much easier for me personally), but of course I'm happy to change or remove the wrapping based on any existing conventions in the project .
> 
> > My only other suggestion would be potentially merging the **Approach Instructions** and **Delaying Aircraft** headings together, since they both cover similar content. Maybe they could both fall under **Sequencing**?
> 
> I was intending to add further information (perhaps with diagrams) in a future PR to each of these sections, so I think they're warranted (and help to break up the page), but happy to remove if you still think they're unnecessary.

Good point, thanks mate.

I'd probably support merging the two headings into a single 'Sequencing' heading, since 'Approach Instructions' makes it sound like an instrument approach.  I'd say that the items you've touched on describe different options for sequencing circuit traffic with inbound/outbound traffic rather than items that would be relevant if there's no other traffic in the area.  What are your thoughts on that?

**Comment by oliverfrye** [comment]
> I'd probably support merging the two headings into a single 'Sequencing' heading, since 'Approach Instructions' makes it sound like an instrument approach. I'd say that the items you've touched on describe different options for sequencing circuit traffic with inbound/outbound traffic rather than items that would be relevant if there's no other traffic in the area. What are your thoughts on that?

OK, sounds good. I've removed the subheadings from "Sequencing".

**Review by mattkelly4 — APPROVED** [review]

**Review by LeGiraffe — APPROVED** [review]

**Pull request merged**